src/main/java/org/mule/extension/sftp/internal/stream/AbstractNonFinalizableFileInputStream.java

```diff
@@ -23,6 +22,10 @@
 import org.apache.commons.io.input.ClosedInputStream;
 import org.apache.commons.io.input.ProxyInputStream;
 
+import static org.mule.extension.sftp.internal.util.StreamProxyUtil.getInputStreamFromStreamFactory;
```

Comment:
organize static imports with this layout:
import static org.mule.*
import static com.mulesoft.*
<blank line>
import static java.*
import static javax.*
<blank line>
import static all other imports
<blank line>
import org.mule.*
import com.mulesoft.*
<blank line>
import java.*
import javax.*
<blank line>
import all other imports